src/sample/index.ts

```diff
@@ -6,9 +6,9 @@
  * Copyright 2021 Datadog, Inc.
  */
 
-import * as cdk from "@aws-cdk/core";
 import * as lambda from "@aws-cdk/aws-lambda";
-import { Datadog } from "../lib/index";
+import * as cdk from "@aws-cdk/core";
+import { Datadog } from "../index";
```

Comment:
I think we had a discussion on how this technically is interacting with the typescript files but once this index.ts is compiled to js via `yarn watch` then it is referencing the javascript index file as the entry point. That is the same entry point that the package.json states. I still will end up testing with my cdk demo from the cloud integrations repo to get the full experience of how a customer would interact with the package.